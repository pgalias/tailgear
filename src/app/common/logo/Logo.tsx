import React, { FunctionComponent } from 'react';

export const Logo: FunctionComponent = () => (
  <img
    className="inline-block"
    src="https://via.placeholder.com/150x50?text=put+logo+here"
    alt="No logo yet :)"
    title="No logo yet :)"
  />
);
