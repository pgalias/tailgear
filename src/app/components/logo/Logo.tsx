import React, { FunctionComponent } from 'react';
import { ReactComponent as Zephyrus } from '../../../assets/images/logo.svg';
import styles from './styles.module.css';

export const Logo: FunctionComponent = () => (
  <Zephyrus className={styles.logo} />
);
