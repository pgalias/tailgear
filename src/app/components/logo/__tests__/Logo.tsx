import React from 'react';
import { render, screen } from '@testing-library/react';
import { Logo } from '../Logo';

describe('app::components::Logo', () => {
  test('should render a logo', () => {
    render(<Logo />);

    expect(screen.getByAltText(/no logo yet :\)/i)).toBeInTheDocument();
    expect(screen.getByRole('img')).toMatchSnapshot();
  });
});
