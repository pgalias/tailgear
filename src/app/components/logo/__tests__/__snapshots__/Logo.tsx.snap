// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app::components::Logo should render a logo 1`] = `
<img
  alt="No logo yet :)"
  class="inline-block"
  src="https://via.placeholder.com/150x50?text=put+logo+here"
  title="No logo yet :)"
/>
`;
